api/proto/adscert.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package api;
+
+option go_package = "/cmd/internal/api";
```

Comment:
I *think* this wants it to use the full package name like you'd find in a regular go file (github.com/IABTechLab/adscert/api/proto) although not 100% sure since I only use protos within Google's flavor of build environment :-)

Reply:
updated to use the full module path
issue was where the generated code was being placed but I figured out the compiler options for `protoc`

Reply:
OK, LGTM... thanks!